File: Python/Algoritmos_Orden/Seleccion.py
def selection(a):
    for i in range(len(a)):
        #elemento mas pequeño
        small = i 
        #encuentra el elemento actual
        for j in range(i+1, len(a)):
            if a[small] > a[j]:
                #compara el mas pequeño con el actual
                small = j
        a[i], a[small] = a[small], a[i]

def printArr(a):
    for i in range(len(a)):
        print(a[i], end = " ")

a = [65,26,13,23,12]
print("Arreglo antes de ser ordenado: ")
printArr(a)
selection(a)
print("Arreglo después de ser ordenado: ")
selection(a)
printArr(a)

File: JavaScript/Algoritmos_Orden/seleccion.js
function selection(a) {
    for (let i = 0; i < a.length; i++) {
        let small = i;
        for (let j = i + 1; j < a.length; j++) {
            if (a[small] > a[j]) {
                small = j;
            }
        }
        //Intercambio igual que en Python
        [a[i], a[small]] = [a[small], a[i]];
    }
}

function printArr(a) {
    let salida = "";
    for (let i = 0; i < a.length; i++) {
        salida += a[i] + " ";
    }
    console.log(salida);
}

let a = [65, 26, 13, 23, 12];

console.log("Arrelo antes de ser ordenado: ");
printArr(a);

selection(a);

console.log("Arreglo después de ser ordenado: ");
selection(a);
printArr(a);